src/main/java/hu/oe/nik/szfmv/automatedcar/CarIndexState.java

```diff
@@ -0,0 +1,35 @@
+package hu.oe.nik.szfmv.automatedcar;
+
+/**Indicates whether the car is indexing and in what direction.
+ * @author Team 3 (Dávid Magyar | aether-fox | davidson996@gmail.com)*/
+public enum CarIndexState {
```

Comment:
Nem volt már erre egy enum korábban? Még oda is kommenteztem, hogy létezik olyan állapot is, hogy mindkettő aktív, bár itt nem fog kelleni. Nem lényeges.

Reply:
> Nem teljesen értem ezt a PR-t, a VFB+-t már elfogadtam.

Az a `Team3-dev`-be ment, akkor még nem tudtam, hogy csak a `master` merge-ek érdekelnek, elnézést.

> Nem volt már erre egy enum korábban?

Nem tudom, a packet-ben nem láttam használva semmi ilyen enum-ot sajnos. De ránézek, hátha van valami használatlan enum eldugva valahol.

Reply:
Megnéztem, igen, az `Index`-nek van külön komponense, abban van `Index.IndexStatus`. Kicsit beleírtam, összegyúrtam a kettőt, enyémet kitöröltem. Integrációt vállalom másik csapattal is.